plugins/jobs/src/js/__tests__/JobsRunNow-test.js

```diff
@@ -0,0 +1,23 @@
+import { Observable } from "apollo-link";
+
+jest.mock("data-service");
+const jobsRunNow = require("../JobsRunNow");
+
+describe("Jobs Run Now Action", function() {
+  it("runNowJobGraphql call if made with onItemSelect", function(done) {
+    const runNowJobGraphql = jest.fn().mockReturnValue(
+      new Observable(() => {
+        expect(runNowJobGraphql).toBeCalled();
+        done();
+      })
+    );
+
+    const jobsRunNowAction = jobsRunNow.jobsRunNowAction(
+      "test",
+      runNowJobGraphql
+    );
+
+    jobsRunNow.jobsRunNow$.forEach(() => {});
```

Comment:
What's the purpose of this?

Reply:
Basically you have to listen to the observable for it to actually start as it is lazy. Changed this to subscribe.